README.md

```diff
@@ -28,6 +28,607 @@ sha256sum -c sha256sum.txt 2>&1 | grep OK
 oauth2_proxy-3.2.0.linux-amd64: OK
 ```
 
+2.  Select a Provider and Register an OAuth Application with a Provider
```

Comment:
Bad merge here too, the documentation has been moved into the `docs` folder, please merge you docs changes into that folder